src/os/alert/alertevent.js

```diff
@@ -58,6 +58,18 @@ export default class AlertEvent extends GoogEvent {
      */
     this.dismissDispatcher_ = opt_dismissDispatcher || null;
 
+    /**
+     * @type {?number}
+     * @private
+     */
+    this.handlerTimeoutId_ = null;
```

Comment:
The handler/timeout are specific to the alert popup UI, so I don't think it makes sense to track them on the alert event. The event could be handled by any number of UI's, so its API should not include UI-specific properties. These can be tracked on the alert popup object instead.